actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -527,22 +542,24 @@ private void orgConsumptionMetrics(Request actorMessage) {
         return;
       }
       String channel = (String) rootOrgData.get(JsonKey.HASHTAGID);
-      ProjectLogger.log("channel" + channel);
-      String responseFormat = (String) cache.getData(JsonKey.OrgConsumption, orgId, periodStr);
-      if (StringUtils.isBlank(responseFormat)) {
-        responseFormat = getOrgConsumptionData(actorMessage, periodStr, orgHashId, channel);
-        ProjectLogger.log("Response" + responseFormat);
-        cache.setData(JsonKey.OrgConsumption, orgId, periodStr, responseFormat);
-      }
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgConsumptionMetrics hash tagid value " + channel,
+          LoggerEnum.INFO.name());
+      String responseFormat = getOrgConsumptionData(actorMessage, periodStr, orgHashId, channel);
+      ProjectLogger.log("Response" + responseFormat);
       Response response =
           metricsResponseGenerator(responseFormat, periodStr, getViewData(orgId, orgName));
       sender().tell(response, self());
     } catch (ProjectCommonException e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgConsumptionMetrics ProjectCommonException occured "
+              + e.getMessage(),
+          e);
       sender().tell(e, self());
       return;
     } catch (Exception e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgConsumptionMetrics Exception occured " + e.getMessage(), e);
```

Comment:
From:
"OrganisationMetricsActor:orgConsumptionMetrics Exception occured "

To:
"OrganisationMetricsActor:orgConsumptionMetrics: Generic exception in getting org consumption data: "